### DISCUSSION TITLE
Nokogiri compilation broke a few days ago

### DISCUSSION THREAD

**Discussion by skyzyx** [lowlighter/metrics#1678] [Q&A]
Log file is being written to `/var/lib/gems/3.1.0/extensions/x86_64-linux/3.1.0/nokogiri-1.18.1/mkmf.log` during compilation.

**Comment by unlsycn** [comment]
The same problem, full log: https://github.com/unlsycn/unlsycn/actions/runs/12497837064/job/34870930453
